client/session/defaults.go

```diff
@@ -1,12 +1,10 @@
 package session
 
 import (
-	"context"
 	"net/http"
 )
 
-// Defaults returns default context and HTTP client to use by clients that don't
-// need any further configuration.
-func Defaults() (context.Context, *http.Client) {
-	return context.Background(), http.DefaultClient
+// Defaults returns a HTTP client to use by clients that don't need any further configuration.
+func Defaults() *http.Client {
+	return http.DefaultClient
```

Comment:
arguably this doesn't make much sense anymore

Reply:
did a small refactor here. I don't like to use the DefaultClient so I created a separate one and moved it to "client/httpclient.go"